src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -112,7 +112,7 @@ foam.CLASS({
 
                 // Render title
                 .start('p').addClass(self.myClass('title'))
-                  .translate([wizardlet.id, 'title'], wizardlet.title)
+                  .translate(wizardlet.capability.id+'.name')
```

Comment:
I think: ['wizardlet', wizardlet.id, 'title']

Reply:
Right now `wizardlet.id` will be something like `capability,UUID-STRING`, which will become `capability,UUID-STRING,title` after the code that's currently in place. Not all wizardlets are capability wizardlets, so I made determining the prefix the wizardlet's responsibility so the prefix could be `capability`. That said, if all our IDs are UUIDs then the prefix probably isn't needed at all.

@kgrgreer 's solution works, but if we translate the capability title outside of a wizardlet we might end up creating duplicate translation entries. (ex: `capability.UUID-STRING.title` and also `wizardlet.UUID-STRING.title`)

`wizardlet.capability.id+'.name'` should not be used because not all wizardlets are capability wizardlets.